internal/services/compute/shared_image_gallery_resource.go

```diff
@@ -53,6 +55,64 @@ func resourceSharedImageGallery() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"sharing_profile": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"permission": {
+							Type:     pluginsdk.TypeString,
+							Required: true,
+							ForceNew: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								string(galleries.GallerySharingPermissionTypesCommunity),
+							}, false),
+						},
+
+						"community_gallery_info": {
```

Comment:
```suggestion
						"community_gallery": {
```
Same here, as mentioned above, should we shorten this to just `community_gallery`?